README.md

```diff
@@ -68,6 +68,7 @@ HeaderGenerator randomly generates realistic browser headers based on specified
 * [HeaderGenerator](#HeaderGenerator)
     * [`new HeaderGenerator(options)`](#new_HeaderGenerator_new)
     * [`.getHeaders(options, requestDependentHeaders)`](#HeaderGenerator+getHeaders)
+    * [`orderHeaders(headers, [order])`](#HeaderGenerator+orderHeaders)
```

Comment:
It's missing the leading dot. That's weird. Did you build the docs @szmarczak or did you add it manually?

Reply:
Manually 😅 Didn't know there's a doc builder. Lemme fix